lib/aur-chroot

```diff
@@ -296,16 +312,39 @@ if (( build )); then
     )
 
     # create makepkg_user
-    $AUR_PACMAN_AUTH systemd-sysusers --root "$directory"/root --replace=/usr/lib/sysusers.d/build.conf - \
-        < <(printf "u %q $(id -u) \"makepkg user\" /build /bin/bash\n" "$makepkg_user")
+    systemd-sysusers --root "$directory"/root --replace=/usr/lib/sysusers.d/build.conf - \
+        < <(printf "u %q %s \"makepkg user\" /build /bin/bash\n" "$makepkg_user" "$AUR_CHROOT_UID")
 
     # allow makepkg_user to install packages (e.g. --syncdeps)
-    $AUR_PACMAN_AUTH tee "$directory/root/etc/sudoers.d/makepkg_user" \
+    tee "$directory/root/etc/sudoers.d/makepkg_user" \
         < <(printf '%q ALL = NOPASSWD: /usr/bin/pacman' "$makepkg_user") \
         1> /dev/null
-    $AUR_PACMAN_AUTH chmod 440 "$directory/root/etc/sudoers.d/makepkg_user"
+    chmod 440 "$directory/root/etc/sudoers.d/makepkg_user"
+
+    {
+        printf 'makepkg'
```

Comment:
Fwiw:
```sh
printf 'makepkg %s || exit\n' "${*@Q}"` > "$directory"/root/aur-chroot--helper
```

Reply:
There had to be a way, thanks for showing me! :)